### ISSUE TITLE
 Cannot read properties of undefined when running sample

### ISSUE THREAD

**Issue by asgerhallas** [run-llama/LlamaIndexTS#570]
When running this sample: https://github.com/run-llama/LlamaIndexTS/blob/c8396c5a3c16432cc3537006c7b844037b820b17/apps/docs/docs/modules/agent/multi_document_agent.mdx

I get the following error: 

```
Added user message to memory!: Tell me the differences between Brazil and Canada economics?
C:\workspace\sandbox\llama-1\node_modules\llamaindex\index.js:4007
                    name: tool.metadata.name,
                               ^

TypeError: Cannot read properties of undefined (reading 'metadata')
    at C:\workspace\sandbox\llama-1\node_modules\llamaindex\index.js:4007:32
    at Array.map (<anonymous>)
    at C:\workspace\sandbox\llama-1\node_modules\llamaindex\index.js:4006:39
    at Generator.next (<anonymous>)
    at asyncGeneratorStep$O (C:\workspace\sandbox\llama-1\node_modules\llamaindex\index.js:3781:28)
    at _next (C:\workspace\sandbox\llama-1\node_modules\llamaindex\index.js:3799:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v20.10.0
```

Can somebody help me? :)

**Comment by EmanuelCampos** [comment]
Fixed the example

TLDR;  remove `storageContext` from ObjectIndex

https://github.com/run-llama/LlamaIndexTS/blob/14d5a1458c072518135b0dd00db4dfe37a4122e7/examples/agent/multi_document_agent.ts#L125-L132

**Comment by asgerhallas** [comment]
Thanks!!
